src/web/index.ts

```diff
@@ -0,0 +1,96 @@
+/* eslint no-bitwise: ["error", { "allow": ["~", "&", ">>"] }] */
+/* global navigator, window */
+
+import CborReader from 'cbor-js';
+
+import { WebCryptoModule, LegacyCryptor, AesCbcCryptor } from '../crypto/modules/WebCryptoModule/webCryptoModule';
+import { stringifyBufferKeys } from '../core/components/stringify_buffer_keys';
+import { CryptorConfiguration } from '../core/interfaces/crypto-module';
+import { PubNubFileParameters, PubNubFile } from '../file/modules/web';
+import { makeConfiguration } from '../core/components/configuration';
+import { PubNubConfiguration, setDefaults } from './configuration';
+import TokenManager from '../core/components/token_manager';
+import { PubNubMiddleware } from '../transport/middleware';
+import { WebTransport } from '../transport/web-transport';
+import { decode } from '../core/components/base64_codec';
+import Crypto from '../core/components/cryptography';
+import WebCryptography from '../crypto/modules/web';
+import { PubNubCore } from '../core/pubnub-common';
+import Cbor from '../cbor/common';
+
+/**
+ * PubNub client for browser platform.
+ */
+export default class PubNub extends PubNubCore<ArrayBuffer | string, PubNubFileParameters, PubNubFile> {
+  /**
+   * Data encryption / decryption module constructor.
+   */
+  static CryptoModule = WebCryptoModule;
+
+  constructor(configuration: PubNubConfiguration) {
+    const configurationCopy = setDefaults(configuration);
+    const platformConfiguration = { ...configurationCopy, sdkFamily: 'Nodejs', PubNubFile };
```

Comment:
Shouldn’t it be ‘web’ instead of `‘Nodejs’`?

> sdkFamily: 'Web'

Reply:
Oh shoot, probably didn't save when fixed it once. Thank you for noticing!